aiozk/session.py

```diff
@@ -72,8 +73,13 @@ def __init__(self, servers, timeout, retry_policy, allow_read_only, read_timeout
         await self.state.wait_for(*safe_states, loop=self.loop)
 
     async def start(self):
+        if self.started:
+            return
```

Comment:
What if everything was called but we didn't reach safe state?
See `await self.ensure_safe_state()` below.